lib/core/util.c

```diff
@@ -224,7 +224,7 @@ static h2o_iovec_t build_redis_value(h2o_iovec_t session_data)
 
 #undef BASE64_LENGTH
 
-static void redis_resumption_on_get(redisReply *reply, void *_accept_data)
+static void redis_resumption_on_get(redisReply *reply, void *_accept_data, const char *errstr)
```

Comment:
If I recall correctly, `on_redis_resumption_get_failed` is introduced with zero timeout because  `h2o_socket_ssl_resume_server_handshake` or something like that is not reentrant.